### ISSUE TITLE
Error: 'module' object has no attribute 'module', Why?

### ISSUE THREAD

**Issue by dengdan** [dmlc/mxnet#2582]
As described by the title:
```
>>> import mxnet as mx
>>> mx.mod
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'mod'
>>> mx.mod.Module
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'mod'
>>> mx.module
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'module'
>>> 
```

**Comment by tqchen** [comment]
try update your mxnet to latest version

**Comment by dengdan** [comment]
Oh, I see. My mxnet was installed months ago. The 'module' was added long after my installation. Thank you.

**Comment by dengdan** [comment]
fixed #2582
